src/main.py

```diff
@@ -123,15 +126,14 @@ def edit_profile():
 
 
 @app.route("/edit-subscription", methods=['POST', 'GET'])
+@login_required
 def edit_subscription():
-    global current_user_email
-
     form = EditSubscriptionForm()
-    if not current_user_email:
+    if not current_user.is_authenticated:
```

Comment:
isn't this guard redundant while using login_required decorator?

Reply:
Yes, it is. Made requested changes